src/main/java/org/jabref/gui/JabRefFrame.java

```diff
@@ -1316,6 +1323,31 @@ public void execute() {
         }
     }
 
+    private class CloseOthersDatabaseAction extends SimpleCommand {
```

Comment:
Maybe you can add a utility method in ActionHelper to check if there are multiple databases present? In that case the entry for this could be disabled if not.

Reply:
Do you mean that I only need to disable the menu items when there is only one databese opened but no need to hide these items? 
Just like the following is OK?

![image](https://user-images.githubusercontent.com/55199983/83330920-169f0a00-a2c5-11ea-9e1e-82e492a60ccc.png)